zebra-chain/src/parameters/network_upgrade.rs

```diff
@@ -72,6 +72,12 @@ pub(crate) const TESTNET_ACTIVATION_HEIGHTS: &[(BlockHeight, NetworkUpgrade)] =
 #[derive(Copy, Clone, Debug, Eq, Hash, PartialEq)]
 pub struct ConsensusBranchId(u32);
 
+impl From<ConsensusBranchId> for u32 {
```

Comment:
We should do From impls more often :-)

Reply:
traits in general are strong and my friend

Reply:
I love `From` impls